Dshop/apps/users/models.py

```diff
@@ -1,28 +1,30 @@
-from django.db import models
 from django.contrib.auth.models import User
+from django.db import models
+from django.utils.translation import gettext_lazy as _
 
 
 class Country(models.Model):
     name = models.CharField(max_length=100)
     code = models.CharField(max_length=2)
 
     class Meta:
-        verbose_name_plural = 'Countries'
+        verbose_name = _('Country')
+        verbose_name_plural = _('Countries')
 
     def __str__(self):
         return self.name
 
 
 class CustomUser(models.Model):
-    user = models.OneToOneField(User, on_delete=models.CASCADE)
-    first_name = models.CharField(max_length=100)
-    last_name = models.CharField(max_length=100)
-    address = models.CharField(max_length=150)
+    user = models.OneToOneField(User, on_delete=models.CASCADE, related_name='customusers', verbose_name=_('User'))
+    first_name = models.CharField(max_length=100, blank=True, verbose_name=_('First name'))
```

Comment:
Why blank=True? It is required. This goes for all the fields. 
This task was to add API to manage users, not to change business logic around user models. 
Please remove all the blank=True introduced.